tests/test_app.py

```diff
@@ -152,6 +152,20 @@ def test_patch_dataset(self, create_app, caplog):
           image_url=IMAGE_URL)
       assert "SUCCESS" in caplog.text
 
+  def test_patch_model(self, create_app, caplog):
+    with caplog.at_level(logging.INFO):
+      create_app.patch_model(
+          model_id=CREATE_MODEL_ID,
+          action='merge',
```

Comment:
I hope you have tested with other actions like "overwrite" and "remove", ignore if you have tested it locally.

Reply:
Yes, I have done it